contracts/contracts/PreConfCommitmentStore.sol

```diff
@@ -239,12 +239,13 @@ contract PreConfCommitmentStore is IPreConfCommitmentStore, Ownable2StepUpgradea
             "Invalid commitment digest"
         );
 
-        address commiterAddress = commitmentDigest.recover(commitmentSignature);
+        address committerAddress = commitmentDigest.recover(commitmentSignature);
 
         address winner = blockTracker.getBlockWinner(blockNumber);
         require(
-            (msg.sender == winner && winner == commiterAddress) ||
-                msg.sender == bidderAddress,
+            winner == committerAddress && 
```

Comment:
Is this new logic intended to be equivalent but more readable? or a bug fix?

Reply:
Yes, it's a bug fix, you should be able to open only winning commitments

Reply:
I did this bcs of this discussion: https://primevworkspace.slack.com/archives/C04S164MN9F/p1722421640816879